### DISCUSSION TITLE
QCarousel - Next slide on mousewheel

### DISCUSSION THREAD

**Discussion by Mbensler** [quasarframework/quasar#12217] [General - Components / Directives / etc] [answered]
Hello, I don't know if this is the right place to ask, but I'm trying to build something with the Quasar Framework. Specifically I'm trying to make it so the QCarousel component goes to the next slide on mousewheel scroll, instead of having to click on next slide buttons. Can anyone tell me if its possible to create this behaviour? 🙂

**Comment by pdanpdan** [comment]
make a codepen, please

**Comment by yusufkandemir** [comment, marked as answer]
https://codepen.io/yusufkandemir/pen/yLPyQLY

I already answered this on our Discord server, and the OP accepted it. So, as a side note to all users, please indicate that you asked this question in multiple channels and link them when possible next time. This way we can avoid duplicate efforts of team/staff/community members.
